pandas/core/frame.py

```diff
@@ -7086,6 +7086,7 @@ def melt(
         var_name=None,
         value_name="value",
         col_level=None,
+        ignore_index=True,
```

Comment:
can you type ( i know nothing else is, but at least this is a start)

Reply:
Of course!